lib/block-supports/typography.php

```diff
@@ -89,18 +89,19 @@ function gutenberg_apply_typography_support( $block_type, $block_attributes ) {
 
 	// Font Family.
 	if ( $has_font_family_support ) {
-		$has_font_family = isset( $block_attributes['style']['typography']['fontFamily'] );
+		$font_family = $block_attributes['style']['typography']['fontFamily'];
 		// Apply required class and style.
-		if ( $has_font_family ) {
-			$font_family = $block_attributes['style']['typography']['fontFamily'];
+		if ( isset( $font_family ) ) { // Legacy case.
 			if ( strpos( $font_family, 'var:preset|font-family' ) !== false ) {
 				// Get the name from the string and add proper styles.
 				$index_to_splice  = strrpos( $font_family, '|' ) + 1;
 				$font_family_name = substr( $font_family, $index_to_splice );
 				$styles[]         = sprintf( 'font-family: var(--wp--preset--font-family--%s);', $font_family_name );
 			} else {
-				$styles[] = sprintf( 'font-family: %s;', $block_attributes['style']['color']['fontFamily'] );
+				$styles[] = sprintf( 'font-family: %s;', $font_family );
 			}
+		} elseif ( isset( $block_attributes['fontFamily'] ) ) { // Current attribute style.
+			$styles[] = sprintf( 'font-family: %s;', $block_attributes['fontFamily'] );
```

Comment:
I still must be doing something wrong, I don't see this applied to the Verse block on the frontend 🤔